ctapipe/coordinates/ground_frames.py

```diff
@@ -200,3 +221,38 @@ def project_to_ground(tilt_system):
     y_projected = y_initial - trans[2][1] * z_initial / trans[2][2]
 
     return GroundFrame(x=x_projected * unit, y=y_projected * unit, z=0 * unit)
+
+
+@frame_transform_graph.transform(FunctionTransform, GroundFrame, GroundFrame)
+def ground_to_ground(ground_coords, ground_frame):
+    return ground_coords
+
+
+@frame_transform_graph.transform(FunctionTransform, GroundFrame, EastingNorthingFrame)
+def ground_to_easting_northing(ground_coords, eastnorth_frame):
+    """
+    convert GroundFrame points into eastings/northings for plotting purposes
+
+    """
+
+    return eastnorth_frame.realize_frame(
+        CartesianRepresentation(
```

Comment:
This looks like it could easier be an `AffineTransform` with a simple, constant transform matrix

Reply:
good point - I didn't actually look to see what transforms are provided by Astropy since it was so simple, but it would be clearer to use one.